### DISCUSSION TITLE
What exactly is the process to install comfyui on my pc

### DISCUSSION THREAD

**Discussion by Dralokonda** [comfyanonymous/ComfyUI#5275] [Q&A]
My laptop is a Dell Vostro Intel Core with a intel xe graphics card, 134217728 kolobytes, 4 cores and 8 logical processors with 8 gb ram. I was recently recommended to install comfyui on my pc, however, ive run into issues. Please keep in mind, I am not an IT person, I most do 3d modelling in autodesk maya so i don't have as much knowledge on programming, prompts and never had AI installed on my pc before.

On some installing tutorials theres no instructions on installation for cpu, since i don't have nvidia graphics card for gpu.

I don't whether i must already have stable diffusion installed on my pc first before installing comfyui and how to install it for cpu only

I keep running into problems with my prompt when installing miniconda for the anaconda prompt

Should i uninstall python, miniconda and everything else first and delete all prompt files so i can start again fresh?

**Comment by ltdrdata** [comment]
Have you tried this?
https://github.com/comfyanonymous/ComfyUI#intel-gpus
